src/community/advanced-data-card.tsx

```diff
@@ -57,46 +59,51 @@ const CardContent: React.FC<CardContentProps> = ({
     subtitleLinesMax,
     description,
     descriptionLinesMax,
+    href,
+    ariaLabel,
 }) => {
     const {textPresets} = useTheme();
 
     return (
         <Stack space={4}>
             {headline}
-            <Stack space={4}>
-                {pretitle && (
-                    <Text2
-                        color={vars.colors.textPrimary}
-                        truncate={pretitleLinesMax}
-                        as={pretitleAs}
-                        regular
-                        hyphens="auto"
-                    >
-                        {pretitle}
-                    </Text2>
-                )}
-                <Text
-                    mobileSize={18}
-                    mobileLineHeight="24px"
-                    desktopSize={20}
-                    desktopLineHeight="28px"
-                    truncate={titleLinesMax}
-                    weight={textPresets.cardTitle.weight}
-                    as={titleAs}
-                    hyphens="auto"
-                >
-                    {title}
-                </Text>
+            <Touchable
+                tabIndex={0}
+                maybe
+                className={classNames(styles.touchableAccessibility)}
+                as="a"
+                href={href}
+                aria-label={ariaLabel}
+            >
+                <div />
```

Comment:
The empty div is because the touchable must have a child, and the reason the touchable is there is so the touchable does not interfere with the screen reader's reading order.

The touchable is as a link, because it should be read as such on the screen reader.